core/src/main/resources/hudson/tasks/Fingerprinter/FingerprintAction/index.jelly

```diff
@@ -36,18 +36,20 @@ THE SOFTWARE.
         <l:icon class="icon-fingerprint icon-xlg"/>
         ${%Recorded Fingerprints}
       </h1>
-      <table class="fingerprint-in-build sortable bigtable">
-        <tr>
-          <th initialSortDir="down">${%File}</th>
-          <th>${%Original owner}</th>
-          <th>${%Age}</th>
-        </tr>
+      <table class="fingerprint-in-build sortable jenkins-table">
+        <thead>
+          <tr>
+            <th initialSortDir="down">${%File}</th>
+            <th>${%Original owner}</th>
+            <th>${%Age}</th>
+          </tr>
+        </thead>
         <j:forEach var="e" items="${it.fingerprints}">
           <j:set var="f" value="${e.value}" />
           <tr>
             <td>
               <a href="${rootURL}/fingerprint/${f.hashString}/">
```

Comment:
Looks great to me :) Only thing I've spotted is this link is missing `class="jenkins-table__link"`

Reply:
Good catch!